assets/js/base/components/cart/quantity-selector/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const QuantitySelector = ( { className, quantity, onChange } ) => {
+	const classes = classNames( 'wc-block-quantity-selector-input', className );
```

Comment:
> I wonder if we should follow similar a similar schema we've used for css classes of {selector}__{target} (or something like that).

Still getting my head around BEM and how we're using it, will update classes along the lines you've suggested (note some components have changed name so will take that into account).

> Fwiw, I actually think we should be using wc-component as the prefix for components and wc-block only for things that serve directly as the main block container. But that ship has sailed for now I think (open to discussion on it though).

Seems sensible. I was assuming we were using `wc-block` for our "module" prefix (like namespace). `wc-component` seems like a good prefix for shared components (i.e. stuff in `base/components`; rather than use for components used somewhere in a block (i.e. implementation detail/sub components). Something to discuss, especially in context of sharing components with woo core, wc-admin etc :)

Reply:
Regarding `wc-component` as a prefix, we should take a project-wide decision regarding this and probably #1454 is a good place to discuss it. But until we reach a consensus, I would stick to `wc-block` for everything as we have been doing until now.

One thing that makes me reluctant to use `wc-component` is that there is a package named `@woocommerce/components` which is part of WC Admin, and that prefix seems better suited for that package than for ours.

Reply:
> One thing that makes me reluctant to use wc-component is that there is a package named @woocommerce/components which is part of WC Admin, and that prefix seems better suited for that package than for ours.

That's actually _exactly_ why I want to use `wc-component` here.  One of the criteria I think would be components that will be shared and eventually even moved into the `@woocommerce/components` package.  Using the css prefix would be a good signal for that.